contest/contributions/malishev-covid19.yml

```
@@ -0,0 +1,10 @@
+# Contribution's title, author, abstract
+title: Realtime interactive global distribution of 2019-nCov showing cases deaths and cases in last 15 days 
```

Align title style (mainly title-case)
```suggestion
title: Real-time Interactive Global Distribution of 2019-nCov: Cases, Deaths and Cases in Last 15 Days 
```